### PR TITLE
(2) Fix/doc dataset #449 + Docstring parsing #336 

### PR DESCRIPTION
**Pull request by AxelRinghausen**
* **What kind of changes does this PR introduce?** 
- [ ] Bug fix
- [ ] new features
- [ ] performance 
- [x] docs update

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes:
- [x] No

* **Other information**:

### CODE REVIEW

File: dessia_common/forms.py [hunk 1]
```diff
@@ -308,25 +308,24 @@ def generate_from_text(cls, stream: StringFile):
 
     def add_standalone_object(self, object_: StandaloneBuiltinsSubobject):
         """
-        Add a standalone object to object_list.
-
-        It doesn't return anything, hence, API will update object when computing from frontend.
+        Adds a standalone object to object_list.
```
Comment (GhislainJ) [inline dessia_common/forms.py:311]: We discussed with Tanguy, and it looks like convention sais the verb should not have an "s" at the end

File: dessia_common/forms.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """
-Module that aims to list all possibilities of data formats offered by Dessia.
-
+A module that aims to list all
```
Comment (GhislainJ) [inline dessia_common/forms.py:2]: Should this be a one-liner ?

File: dessia_common/exports.py [hunk 3]
```diff
@@ -50,8 +51,9 @@ def __init__(self, selector: Optional[str], extension: str, method_name: str, te
         self.args = args
 
     def to_dict(self):
-        return {"selector": self.selector, "extension": self.extension, "method_name": self.method_name,
-                "text": self.text, "export_name": self.export_name, "args": self.args}
+        return {"selector": self.selector, "extension": self.extension,
```
Comment (GhislainJ) [inline dessia_common/exports.py:54]: Do you still use the 80 char max line length rule ? We decided to set it to 120 now

**Review by GhislainJ — CHANGES_REQUESTED** [review]

File: dessia_common/exports.py (hunk 3 above)
Comment (AxelRinghausen) [reply dessia_common/exports.py:54]: Yes I do.

**Review by GhislainJ — APPROVED** [review]

**Pull request merged**